### PR TITLE
Surprise

### PR DESCRIPTION
**Pull request by Farid212**
Ghost part

### CODE REVIEW

**Comment by Farid212** [comment]
I forget take off "h6" issue and ESLINT:fix but count do please check little bit

File: src/views/LandingPage/Sections/GhostRegis.jsx [hunk 1]
```diff
@@ -0,0 +1,99 @@
...
+  const { classes } = props;
+
+  return (
+    <main className={classes.main}>
+      <CssBaseline />
+      <Paper className={classes.paper}>
+        <Avatar className={classes.avatar}>
+          <LockIcon />
+        </Avatar>
+        <Typography component="h1" variant="h5">
```
Comment (Naturalclar) [inline src/views/LandingPage/Sections/GhostRegis.jsx]: `component` only takes `string`, `func`, or `object`
By default it uses `string`, so you can delete the `component` part
https://material-ui.com/api/typography/

Comment (Farid212) [reply src/views/LandingPage/Sections/GhostRegis.jsx]: ```<Typography variant="h5">``` should look like this?

**Comment by Farid212** [comment]
should I merge it or keep it like this form the moment?

File: src/views/LandingPage/Sections/GhostRegis.jsx (hunk 1 above)
Comment (Naturalclar) [reply src/views/LandingPage/Sections/GhostRegis.jsx]: yup, that looks good :+1:

**Comment by Naturalclar** [comment]
if circleci passes, then it can be merged

**Pull request closed without merging**